docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -1,107 +1,79 @@
 [[_javascript_adapter]]
 === JavaScript adapter
 
-{project_name} comes with a client-side JavaScript library that can be used to secure HTML5/JavaScript applications. The JavaScript adapter has built-in support for Cordova applications.
+{project_name} comes with a client-side JavaScript library called `keycloak-js` that can be used to secure web applications. The adapter also comes with built-in support for Cordova applications.
 
-A good practice is to include the JavaScript adapter in your application using a package manager like NPM or Yarn. The `keycloak-js` package is available on the following locations:
+==== Installation
 
-* NPM: https://www.npmjs.com/package/keycloak-js
-* Yarn: https://yarnpkg.com/package/keycloak-js
+The adapter is distributed in several ways, but we recommend that you install the https://www.npmjs.com/package/keycloak-js[`keycloak-js`] package from NPM:
 
-Alternatively, the library can be retrieved directly from the {project_name} server at `{kc_js_path}/keycloak.js` and is also distributed as a ZIP archive.
-
-
-One important thing to note about using client-side applications is that the client has to be a public client as there is no secure way to store client
-credentials in a client-side application. This makes it very important to make sure the redirect URIs you have configured for the client are correct and as specific as possible.
+[source,bash]
+----
+npm install keycloak-js
+----
 
-To use the JavaScript adapter you must first create a client for your application in the {project_name} Admin Console. Make sure `public`
-is selected for `Access Type`.  You achieve this in Capability config by turning OFF client authentication toggle.
+Alternatively, the library can be retrieved directly from the {project_name} server at `{kc_js_path}/keycloak.js` and is also distributed as a ZIP archive. We are however considering the inclusion of the adapter directly from the Keycloak server as deprecated, and this functionality might be removed in the future.
```

Comment:
The Keycloak release notes say that we may remove this in 22, so I recommend not advertizing it.

Reply:
It turns out that this is quite a bit harder than originally expected and it will likely end up after the 22 release. Unless something has changed I am unaware of. cc @stianst